lib/matplotlib/backend_bases.py

```diff
@@ -2791,6 +2785,12 @@ def mouse_move(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                artists = event.inaxes.hitlist(event)
+                if event.inaxes.patch in artists:
+                    artists.remove(event.inaxes.patch)
+                artists.sort(key=lambda x: x.zorder)
+                if artists:
+                    s += artists[-1].get_zdata(event)
```

Comment:
Why only the last one? Is it a thing to want to have more than one zdata at a time?